europe/README.EUROPE

```diff
@@ -49,6 +49,30 @@ examples of use.
 * nkgrf03vel_realigned_xy.ct2
 * nkgrf03vel_realigned_z.gtx
 
+### Ireland: WGS84/ETRS89 ellipsoidal heights -> OSGM15
```

Comment:
> WGS84/ETRS89

Is this how the Ordnance Survey describes it? While being fairly similar, WGS84 and ETRS89 are not the same thing so I want this to be as precise as possible. It may be that accuracy of the grids are not high enough to make the distinction between the to datums. So some confirmation from the OS docs would be nice.